src/otx/algo/detection/heads/__init__.py

```diff
@@ -2,9 +2,6 @@
 # SPDX-License-Identifier: Apache-2.0
 """Custom head implementations for detection task."""
 
-from .custom_anchor_generator import SSDAnchorGeneratorClustered
-from .custom_atss_head import CustomATSSHead
-from .custom_ssd_head import SSDHead
 from .yolox_head import YOLOXHead
 
-__all__ = ["SSDAnchorGeneratorClustered", "CustomATSSHead", "SSDHead", "YOLOXHead"]
+__all__ = ["YOLOXHead"]
```

Comment:
Why were they removed? If removed, why don't you remove `YOLOXHead`?

Reply:
Theoretically we don't use any registry building models. So we don't need to import modules in __init__.py. I want to delete all but I save YOLOX things because YOLOX still use mmdeploy exporter. At the end of the decoupling project we should remove unnecessary import

Reply:
Agree with that.